docs/components/content/CommonHelpers.vue

```diff
@@ -0,0 +1,82 @@
+<script lang="ts" setup>
+import type { FunctionRef } from "../../src/types"
+const fns: Record<
+  string,
+  {
+    description: string
+    return: string
+    arguments: FunctionRef["arguments"]
+    example?: string
+    tip?: string
+  }
+> = {
+  isBefore: {
+    description:
+      "Returns true if the first date is before the second date, otherwise false.",
+    return: "boolean",
+    arguments: [
+      {
+        name: "inputDate",
+        type: "string | Date",
+      },
+      {
+        name: "dateToCompare",
+        type: "string | Date",
+      },
+    ],
+    example: "isBefore",
+  },
+  isAfter: {
+    description:
+      "Returns true if the first date is after the second date, otherwise false.",
+    return: "boolean",
+    arguments: [
+      {
+        name: "inputDate",
+        type: "string | Date",
+      },
+      {
+        name: "dateToCompare",
+        type: "string | Date",
+      },
+    ],
+    example: "isAfter",
+  },
+  isEqual: {
+    description:
+      "Returns true if the first date is equal to the second date, otherwise false.",
+    return: "boolean",
+    arguments: [
+      {
+        name: "dateLeft",
+        type: "string | Date",
+      },
+      {
+        name: "dateRight",
+        type: "string | Date",
+      },
+    ],
+    example: "isEqual",
+  },
+}
+</script>
+
+<template>
+  <PageSection>
+    <HeadingSection title="Common Helpers" class="text-sky-500" />
```

Comment:
Let’s change "Common Helpers" to just "Helpers":

```suggestion
    <HeadingSection title="Helpers" class="text-sky-500" />
```

Reply:
Done